.github/workflows/scripts/health_check_no.sh

```
@@ -0,0 +1,24 @@
+# .github/workflows/scripts/health_check.sh
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)